testdata/jsight_0.3/ser-to-openapi/req.japi.schema_notation/req.japi.schema_notation.jsight_0.3/jsight_notation_in_directive_type.openapi.todo

```diff
@@ -0,0 +1,86 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "",
+    "version": ""
+  },
+  "components": {
+    "schemas": {
+      "typeWithJsightAnnotation_object": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "integer"
+          }
+        }
+      },
+      "typeWithJsightAnnotation_string": {
+        "type": "string"
+      },
+      "typeWithJsightAnnotation_float": {
+        "type": "number",
+        "format": "float"
```

Comment:
For the float type we do not use the `format` attribute. Only `type` and `example`. Like here:

![image](https://github.com/jsightapi/jsight-api-core/assets/104552116/87cc9379-5884-43db-b30a-da9447735304)

Reply:
Removed the "format" key.